styles/styles.css

```diff
@@ -749,6 +749,15 @@ main .section.narrower > .tab-item.active > .section-container {
   max-width: 45rem;
 }
 
+/* section - large-padded */
+main .section.large-padded {
```

Comment:
These paddings should be applied to `section-container` level. `section` by itself should occupy the full space.

Reply:
Fixed